build/dependency-info.sh

```diff
@@ -23,6 +23,7 @@
 #   SPARK_VER - The version of spark
 
 # Parse cudf and spark dependency versions
+set -e
```

Comment:
The same needs to happen in `build/build-info` and the corresponding `failonerror` pom change is needed in the top-level pom's use of the antrun plugin.

Reply:
> The same needs to happen in `build/build-info` and the corresponding `failonerror` pom change is needed in the top-level pom's use of the antrun plugin.

@jlowe Thanks very much for reminding me, updated.

Reply:
Needs 2020 copyrights